modules/widgets/twitter-timeline.php

```diff
@@ -82,6 +82,16 @@ public function admin_scripts( $hook ) {
 	 * @param array $instance Saved values from database.
 	 */
 	public function widget( $args, $instance ) {
+		if ( $this->is_widget_id_deprecated( $instance['type'] ) ) {
```

Comment:
I don't think we need `is_widget_id_deprecated at all; can we just check `if ( 'asdf' === $instance['type'] && time() > 1532563200 ) {`?

Reply:
I was on the fence about this. A method would be self-explanatory, but an inline comment would be pretty explanatory too. 😄

Reply:
Fixed in ce7f10a